src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,68 @@
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+import React from 'react';
+import { getSearchWith } from '../utils/searchHelper';
+
+enum Sex {
+  male = 'm',
+  female = 'f',
+}
```

Comment:
move enum to separate file
```suggestion
enum Sex {
  Male = 'm',
  Female = 'f',
}
```